rma_account_unreconciled/views/rma_line_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+
+        <record id="view_rma_line_form" model="ir.ui.view">
+            <field name="name">rma.order.line.view.form</field>
+            <field name="model">rma.order.line</field>
+            <field name="inherit_id" ref="rma.view_rma_line_form" />
+            <field name="arch" type="xml">
+                <div name="button_box" position="inside">
+                    <field name="unreconciled" invisible="1" />
+                    <button
+                        type="object"
+                        string="Unreconciled Journal Items"
+                        name="action_view_unreconciled"
+                        groups="account.group_account_manager"
+                        class="oe_stat_button"
+                        icon="fa-gears"
+                        attrs="{'invisible':[('unreconciled', '=', False)]}"
+                    >
+                    </button>
+                    <button
+                        type="object"
+                        string="Reconcile"
+                        name="action_open_reconcile"
+                        groups="account.group_account_manager"
+                        class="oe_stat_button"
+                        icon="fa-link"
+                        attrs="{'invisible':[('unreconciled', '=', False)]}"
```

Comment:
Can you change it to be the billing user?

Reply:
Done